pages/api/users/index.js

```diff
@@ -13,7 +13,10 @@ export default async function handler(req, res) {
 
   await connectMongo();
 
-  const fileProfiles = findAllBasic();
+  let fileProfiles = findAllBasic();
+  const initialRandomUsers = req?.query?.initialRandomUsers == 'true';
+
+  const keywords = req.query?.keywords?.split(",") || [];
```

Comment:
does this need to be an array or should be treated a a whole string?

Reply:
String for comparing usernames or names and array for comparing tags